1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -107,11 +116,75 @@ function convert_is_to_slave() {
     <nodeProperties/>
   </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
   "
+  echo $(expand_template "is/${name}" "${template}")
+}
+
+# convert_istag_to_slave converts the OpenShift imagestreamtag to a Jenkins
+# Kubernetes Plugin slave configuration.
+function convert_istag_to_slave() {
+  [ -z "$oc_cmd" ] && return
+  local name=$1
+  local template="
+  <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
+    <inheritFrom></inheritFrom>
+    <name>{{.metadata.name}}</name>
+    <instanceCap>5</instanceCap>
+    <idleMinutes>0</idleMinutes>
+    <label>${name}</label>
```

Comment:
Also clearly not your invention but having these inside of real `.xml` files for syntax highlighting and easy inspection, with this function using an `.xsl` to template it or at least something like `xmlstarlet` for `xpath` editing if the XSL approach is too heavy-weight.